README.md

```diff
@@ -36,35 +41,44 @@ docker build -f Dockerfile-zeth -t zeth-dev .
 # Start the zeth development container
 docker run -ti -p 50051:50051 --name zeth zeth-dev:latest
 
-## All the commands below are ran in the docker container
+# All the commands below are run in the docker container
 # Configure your environment
 . ./setup_env.sh
 
 # Compile the proving server
 mkdir build
 cd build
 cmake .. [<flags (see below)>]
-## (optional) Run the tests
-make check # Builds and run the tests (once the tests are built, calling "make test" suffices to execute them)
-## Compile
+# Compile all libraries and tools, including the prover_server
 make
+# (optional) Run the unit tests
+make test
+# (optional) Run the all tests (unit tests, syntax checks, etc)
+make check
 
-# Start the proving server
-prover_server
+# Start the prover_server process
+src/prover_server
 ```
 
-Note: *flags* in the cmake command may include `-DCMAKE_BUILD_TYPE=Release` for
-an optimized build.
+Note: By default, the prover_server generates a key at startup.  Flags can be
```

Comment:
Please use single spaces after punctuation `.`
i.e. `startup.<space><space>Flags` -> `startup.<space>Flags`

Reply:
prover_server -> `prover_server` (let's use `` since prover_server is a binary/command here)

Reply:
changed